Task_1_5_1/src/main/java/ru/nsu/zhao/Element.java

```diff
@@ -0,0 +1,30 @@
+package ru.nsu.zhao;
+
+/**
+ * 抽象基类，表示所有 Markdown 元素。
+ */
+public abstract class Element {
```

Comment:
Hello, Zhao. This class can be changed to interface. hashCode and equals can be removed.